tests/php/SubsiteTest.php

```diff
@@ -302,7 +302,7 @@ public function testDefaultDomain()
 
         $this->assertEquals($_SERVER['HTTP_HOST'], singleton(Subsite::class)->PrimaryDomain);
         $this->assertEquals(
-            'http://' . $_SERVER['HTTP_HOST'] . Director::baseURL(),
+            'http://' . $_SERVER['HTTP_HOST'],
```

Comment:
In this context Director::baseURL() is simply '/' - so all we're doing here is updating the unit test to not expect a trailing slash

Reply:
Right, the _value_ of the test is correct, but the _intent_ of the test appears to have changed - the test _should_ include the base URL if there is one (there just isn't one in this case) - IMO either a new additional test should be added where a base URL is set and then tested against, or simple a comment can be added which says something along the lines of "If there is a Director::baseURL() value this will also be included" just to make it clear what the functionality should be in that case.

Reply:
Added comment